app/Filament/Resources/InterpretationResource.php

```diff
@@ -49,8 +49,22 @@ public static function table(Table $table): Table
         return $table
             ->columns([
                 Tables\Columns\TextColumn::make('name'),
-                Tables\Columns\TextColumn::make('namespace'),
-                Tables\Columns\TextColumn::make('route'),
+                Tables\Columns\TextColumn::make('context')
+                    ->label(__('Show context'))
+                    ->getStateUsing(fn (Interpretation $record): string => __('Show context').' <span class="sr-only">'.__('for').$record->name.'</span>')
+                    ->html()
+                    ->url(fn (Interpretation $record): string => $record->route_has_params ? route('filament.resources.interpretations.edit', $record) : localized_route($record->route).'#'.Str::slug($record->name))
+                    ->openUrlInNewTab()
+                    ->icon('heroicon-s-external-link')
+                    ->iconPosition('after'),
+                Tables\Columns\IconColumn::make('asl')
+                    ->getStateUsing(fn (Interpretation $record): bool => $record->getTranslation('video', 'ase', false) !== '')
+                    ->boolean()
+                    ->label('ASL Video'),
+                Tables\Columns\IconColumn::make('lsq')
+                    ->getStateUsing(fn (Interpretation $record): bool => $record->getTranslation('video', 'fcs', false) !== '')
+                    ->boolean()
+                    ->label('LSQ Video'),
```

Comment:
These don't appear to be accessible. The SVGs are marked as `role="presentation" aria-hidden="true"`.